include/circt/Dialect/FIRRTL/FIRRTLDeclarations.td

```diff
@@ -10,8 +10,11 @@
 //
 //===----------------------------------------------------------------------===//
 
+def HasCustomSSAName : DeclareOpInterfaceMethods<OpAsmOpInterface,
```

Comment:
Could this just be added to FIRRTLOp then all ops would get it?

Reply:
Does that also mean each FIRRTL op will have to have its own getAsmResultNames? Then, there will be a large duplication in FIRRTLOps.cpp...